### PR TITLE
More clean up.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

Resolving more DeepScan issues.

### CODE REVIEW

File: examples/jsm/loaders/LogLuvLoader.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ UTIF.decode = function ( buff, prm ) {
 	var bin = id == 'II' ? UTIF._binLE : UTIF._binBE;
 	bin.readUshort( data, offset ); offset += 2;
 
-	var ifdo = bin.readUint( data, offset ); offset += 4;
+	var ifdo = bin.readUint( data, offset );
```
Comment (mrdoob) [inline examples/jsm/loaders/LogLuvLoader.js:58]: This one seems suspicious... 🤔

Comment (Mugen87) [reply examples/jsm/loaders/LogLuvLoader.js:58]: It's not necessary to update `offset` at this point since there is no access to the variable anymore.

Comment (mrdoob) [reply examples/jsm/loaders/LogLuvLoader.js:58]: Okay! 👍

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
The majority of remaining DeepScan issues are located in `examples/jsm/libs` and `editor/js/libs`.

It seems DeepScan allows to define an exclusion list so it's possible to exclude certain files form the analysis. However, only the project owner can apply such a configuration.

https://deepscan.io/docs/guides/get-started/excluding-files-and-rules

Do you think it's worth having a look?

**Pull request merged**